client/run-siege

```diff
@@ -65,13 +139,16 @@ def run_siege(options):
         subprocess.check_output(['bash', '-c', current_cmd])
```

Comment:
I think setting shell=True would allow you to just run current_cmd, but I don't know which is more secure :|